### ISSUE TITLE
BTK Cargo Drop

### ISSUE THREAD

**Issue by Fank** [EpochModTeam/DayZ-Epoch#1044]
What about this?
http://www.armaholic.com/page.php?id=10834

I think this will be nice for 1.0.5
I haven't tested it i only have seen them

**Comment by worldwidesorrow** [comment]
The cargo drop system described by Vampire above has been added for 1.0.7.

**Comment by worldwidesorrow** [comment]
I ended up having to remove this feature because of a duping issue involving other players in the vehicle.

**Comment by SmokeyBR** [comment]
this feature is pretty cool not sure if all that usefull but installed into my server but you probably dont if you are willing to put some more work into it id recommenced putting another check to allow cargo drop only if theres one player inside vehicle
`count (crew _vehicle)) == 1` maybe this would work

 and letting cargo drop be used only once unless the player "recharges" for another cargo drop, could be done on fuel stations, not sure how to do that.

**Comment by worldwidesorrow** [comment]
The crew count check was considered, but I didn't really care for it. I might revisit this later when I have time. For now it is low priority.
